definitions/checks/check_epel_repository.rb

```diff
@@ -0,0 +1,19 @@
+class Checks::CheckEpelRepository < ForemanMaintain::Check
+  metadata do
+    label :check_epel_repository
+    description 'Check if EPEL repoistory enabled on system'
+    tags :pre_upgrade
+    confine do
+      feature(:downstream)
+    end
+  end
+
+  def run
+    assert(!epel_enabled?, 'System is subsribed to EPEL repository')
+  end
+
+  def epel_enabled?
+    system_repos = execute("yum repolist enabled -d 6 -e 0| grep -E 'Repo-baseurl|Repo-id'")
+    return true if system_repos.to_s.include?('epel')
```

Comment:
@upadhyeammit ,
No need to return `true` explicitly. 
As `include?` method returns a `boolean` value, please mention only `system_repos.to_s.include?('epel')`.

Reply:
@kgaikwad changes done as requested. Thank You!